packages/cli/src/helpers.ts

```diff
@@ -25,3 +32,29 @@ export async function getWallet(): Promise<IWallet> {
 export function getDIDProvider(): IDIDProvider {
   return didProvider;
 }
+
+export function getCredentialProvider(): ICredentialProvider {
+  return credentialProvider;
+}
+
+export async function selectCredential() {
+  const credentials = await credentialProvider.getCredentials();
```

Comment:
would be cool if the credentialProvider could just iterate over a folder so I could throw different ones easily as I'm testing